extensions/sql-database-projects/package.json

```diff
@@ -280,6 +280,11 @@
           "command": "sqlDatabaseProjects.importDatabase",
           "when": "connectionProvider == MSSQL && nodeType && nodeType == Database && mssql:engineedition != 11",
           "group": "export"
+        },
+        {
+          "command": "sqlDatabaseProjects.importDatabase",
+          "when": "connectionProvider == MSSQL && nodeType && nodeType == Server && mssql:engineedition != 11",
```

Comment:
the flow for server asks for picking one database  - we have validated that right?

Reply:
Yes, it does ask for a database name (amongst the options within that server).